README.md

```diff
@@ -1,142 +1,136 @@
-  - Use `serverless offline start`
-- **Support for unit tests**
-  - Run `npm test` to run your tests
-- **Sourcemaps for proper error messages**
-  - Error message show the correct line numbers
-  - Works in production with CloudWatch
-- **Lint your code with ESLint**
-- **Add environment variables for your stages**
-- **No need to manage Webpack or Babel configs**
+- User registration
+- User authentication
+- Adding hackathon ideas
+- Editing hackathon ideas
+- Providing users created hacks listing
+- Join team to work on hacks
 
----
+## Target Audience
 
-### Demo
+This backend app is targeted at people who needs to connect their frontend client to be able to perform CRUD operation for their hackathons within their organizatios.
 
-A demo version of this service is hosted on AWS - [`https://z6pv80ao4l.execute-api.us-east-1.amazonaws.com/dev/hello`](https://z6pv80ao4l.execute-api.us-east-1.amazonaws.com/dev/hello)
+## Tech Stack
 
-And here is the ES7 source behind it
+- MongoDB
+- AWS API Gateway
+- AWS Lambda
+- AWS Cognito Authentication
+- Serverless Framework
 
-``` javascript
-export const hello = async (event, context) => {
-  return {
-    statusCode: 200,
-    body: JSON.stringify({
-      message: `Go Serverless v1.0! ${(await message({ time: 1, copy: 'Your function executed successfully!'}))}`,
-      input: event,
-    }),
-  };
-};
+## Application Architecture Diagram
 
-const message = ({ time, ...rest }) => new Promise((resolve, reject) =>
-  setTimeout(() => {
-    resolve(`${rest.copy} (with a delay)`);
-  }, time * 1000)
-);
-```
+![diagram](./docs/architectureDiagram.png)
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
 
-### Upgrading from v1.x
+1. **git clone git@github.com:hackthehack/hack-zero-v2-backend.git**
 
-We have detailed instructions on how to upgrade your app to the v2.0 of the starter if you were using v1.x before. [Read about it here](https://github.com/AnomalyInnovations/serverless-nodejs-starter/releases/tag/v2.0).
+2. cd into the directory and run **npm install**
 
-### Requirements
+3. Run a local server by running '**sls offline start**'
 
-- [Install the Serverless Framework](https://serverless.com/framework/docs/providers/aws/guide/installation/)
-- [Configure your AWS CLI](https://serverless.com/framework/docs/providers/aws/guide/credentials/)
+### Prerequisites
 
-### Installation
+1. Must have NodeJS runtime installed
+
+2. Install serverless framework by running **npm install -g serverless**
+
+### Installing
 
 To create a new Serverless project.
 
-``` bash
+```bash
 $ serverless install --url https://github.com/AnomalyInnovations/serverless-nodejs-starter --name my-project
 ```
 
 Enter the new directory
 
-``` bash
+```bash
 $ cd my-project
 ```
 
 Install the Node.js packages
 
-``` bash
+```bash
 $ npm install
 ```
```

Comment:
these instructions aren't really correct are they?

Reply:
Other sections also look they aren't really up to date to with this project vs the bootstrapped readme.

Reply:
Removed **Installation** section of the README to avoid confusion that section was for creating a serverless project, nothing to do with this project, hence the removal.